cmd/update-server-version.go

```diff
@@ -0,0 +1,36 @@
+package cmd
+
+import (
+	"github.com/couchbaselabs/cbdinocluster/deployment"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+)
+
+var updateServerVersionCommand = &cobra.Command{
```

Comment:
Rather than being an explicit command, this should be part of the cluster definition.  cbdinocluster is mostly a declarative rather than imperative.  This means that we should add a "ServerImage" property to the Capella NodeGroup definition, and then modify allocate and modify to use those values to apply the appropriate changes.  Note that for Capella, all node-groups must have the same value or an error is thrown (you can see how we handle that in the beginning of the allocate/modify commands).  It will be necessary to check with the server team to see how you can allocate a cluster with an upfront server image as well.